src/clients/python/schemas/WebViewResponse.py

```diff
@@ -0,0 +1,43 @@
+# DO NOT EDIT: This file is auto-generated by generate-schema/index.ts
+from typing import Union
+import msgspec
+
+class StringResultType(msgspec.Struct, tag_field="$type", tag="string"): 
+    value: str
+
+class BooleanResultType(msgspec.Struct, tag_field="$type", tag="boolean"): 
+    value: bool
+
+class FloatResultType(msgspec.Struct, tag_field="$type", tag="float"): 
+    value: float
+
+class undefined(msgspec.Struct, kw_only=True, omit_defaults=True): 
```

Comment:
Another suspicious `undefined`

Reply:
Thanks! Looks like I got some code gen to clean up..